src/transformers/models/idefics/perceiver.py

```diff
@@ -0,0 +1,147 @@
+"""
```

Comment:
This filw is missing copyright (especially as it seems to come from somewhere else and we need to attribute rights)

Reply:
tagging @VictorSanh

Reply:
code is adapted from https://github.com/lucidrains/flamingo-pytorch which is under an mit license

Reply:
Thank you, Victor. I will edit the file.

Reply:
Still need the actual copyright similar to other modules of the lib

Reply:
yes, I didn't find any examples in our code base - other than 2 places where MIT was used and that's exactly what I copied.

so it'd be awesome if you could point to how much of the license to copy

Reply:
I found one MIT license [here](https://github.com/huggingface/transformers/blob/main/src/transformers/models/mobilebert/modeling_mobilebert.py#L1)

The copyright should be all comments, not in a docstring.